### PR TITLE
Now with docker!

### PR DESCRIPTION
**Pull request by technoe**
I moved the scripts into a scripts folder to make mounting them easier.
Using the compose file moves the .env properties to the compose file. 
Included a small readme update for the files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,10 +106,20 @@ To install a C++ compiler on Windows 10/11, follow these steps:
...
+When running a Mac with Intel hardware (not M1), you may run into _clang: error: the clang compiler does not support '-march=native'_ during pip install.
 
 If so set your archflags during pip install. eg: _ARCHFLAGS="-arch x86_64" pip3 install -r requirements.txt_
 
 # Disclaimer
 This is a test project to validate the feasibility of a fully private solution for question answering using LLMs and Vector embeddings. It is not production ready, and it is not meant to be used in production. The models selection is not optimized for performance, but for privacy; but it is possible to use different models and vectorstores to improve performance.
+
+# Docker
+
+```
```
Comment (thiago-scherrer) [inline README.md:119]:
```suggestion
```sh
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.9
+WORKDIR /root/
+RUN apt-get update && apt-get upgrade -y
+ENV PATH="/root/.local/bin:${PATH}"
+COPY requirements.txt .
+RUN pip3 install --user --upgrade pip
+RUN pip3 install --user -r requirements.txt
```
Comment (thiago-scherrer) [inline Dockerfile:7]: Combine multiple RUN commands into a single one. This reduces the number of image layers and improves build speed.

no-cache-dir avoid caching the downloaded packages.

```suggestion
FROM python:3.10.9

WORKDIR /root/
ENV PATH="/root/.local/bin:${PATH}"

COPY requirements.txt .

RUN apt-get update \
    && apt-get upgrade -y \
    && pip install --upgrade pip \
    && pip install --no-cache-dir -r requirements.txt
```

**Review by thiago-scherrer — COMMENTED** [review]
💯💯💯💯

**Comment by mdeweerd** [comment]
#120 Already using docker - CUDA ready - docker-compose.yml .

**Pull request closed without merging**